lib/sandbox/pmapi.js

```diff
@@ -113,12 +118,24 @@ Postman = function Postman (bridge, execution, onRequest) {
     // add response assertions
     if (this.response) {
         // these are removed before serializing see `purse.js`
-        this.response.to = chai.expect(this.response).to;
+        this.response.to = {not: {}};
+
+        // explicitly assigning chai methods to chai's 'to' and 'to.not'.
+        chainingMethods.forEach(function eachMethod (value) {
```

Comment:
We need to think of a better alternative to this.

Reply:
Yes, the ideal way is to get the list of language chains used by chai (If they expose it explicitly/implicitly). That would ensure no worries while updating chai for postman-sandbox. Though, this change wouldn't break things out of the blue since we are locked to a specific version of chai, I'll look if this can be done with the former way.